src/components/NewAccountPage/NewAccountPage.jsx

```diff
@@ -84,8 +128,26 @@ const StyledButtonAndEye = styled.div`
    display: flex;
    justify-content: flex-end;
    margin-left: auto;
-   margin-top: -125px;
+   margin-top: -60px;
    margin-right: 20px;
+   @media only screen and (min-width: 768px) {
+      margin-top: -6px;
+      position: absolute;
+      margin-left: auto;
+      margin-right: auto;
+   }
+`;
```

Comment:
Replace everything in this styled component with the following code. This fixes the weird spacing with the show/hide password button.

```
position: absolute;
top: 30px;
@media only screen and (max-width: 768px) {
   position: absolute;
   top: 30px;
   right: 40px;
}
```

Reply:
It doesn't fix it on my screen. Maybe we need to come up with a solution that works for both our screen-testing sizes.